src/Raven.Server/Documents/Changes/AbstractChangesClientConnection.cs

```diff
@@ -285,7 +283,23 @@ private async Task WaitForSendTaskAsync(Task sendTask, Stopwatch sp, int message
         }
     }
 
-    private async ValueTask<object> GetNextMessage()
+    protected readonly struct Message : IDisposable
```

Comment:
move it to the bottom of the class, let's avoid creating nested types in the middle of the class